src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,50 @@
 package core.basesyntax;
 
-class Pair {
+import java.util.Objects;
 
+class Pair<T, F> {
+    private T first;
+    private F second;
+
+    private Pair(T first, F second) {
+        this.first = first;
+        this.second = second;
+
+    }
+
+    public T getFirst() {
+        return first;
+    }
+
+    public F getSecond() {
+        return second;
+    }
+
+    public static Pair of(Object o1, Object o2) {
```

Comment:
you should use generics here

Reply:
Good Evening. Corrected as mentioned above.